README.md

```diff
@@ -57,14 +57,14 @@ git commit -m "Create comments migration"
 O mais interessante ainda, seria fazer este commit após a criação da migration e do model. Assim, unificamos um commit para criação tanto do database, quanto do model:
 
 ```
-git commit -m "Create comments migration and Comment model"
+git commit -m "Create comments migration and comment model"
```

Comment:
In this case, we reffering to `Comment` as a model. 😄

Reply:
Okay tho', I'll fix it :D

Reply:
luv u, beach ❤️